### DISCUSSION TITLE
qt teamtalk. New host manager's problems

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2256] [Q&A]
### Description

1: in the context menu of laytist hosts, when i klicked the clear laytist hosts option, my teamtalk hard crashed.
2: the items inside laytist hosts list are just announced as list item for screen readers.
3: When i didn't had any servers saved in my saved servers list, and i klicked the edit option in the context menu, my teamtalk crashed. Context menu options should be grade out when you don't have any server selected.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
Point 1 and 3 are fixed by PR #2253  and #2254 
Don't understand second point. List items are correctly labelled with latest host name.

**Comment by peter1384** [comment]
@CoBC point 2: For example, you have 5 servers in the laytist host list. When you scrole throo them with up and down arrow keys, nvda just says list item, list item, list item.

**Comment by CoBC** [comment]
I can't reproduce

**Comment by jeremyp3** [comment]
@peter1384 
Which version of NVDA are you using?

**Comment by peter1384** [comment]
@CoBC don't worry about the point 2: it's now ok. I don't know how that happened. Just when i tryed that teamtalk with new host manager and i had 1 laytist host from the old host manager remaning, so i cleared everything and started fresh and everything is now ok. Thanks.
